coalib/results/Diff.py

```diff
@@ -352,6 +352,7 @@ def delete_line(self, line_nr):
         """
         Mark the given line nr as deleted. The first line is line number 1.
         """
+        line_nr = 0 if (len(self._file) < line_nr) else line_nr
```

Comment:
Reasons why I don't like this format: 
1) It goes against the normal flow of thought (readability, familiarity).
2) Consistency in the codebase; I don't believe we use it elsewhere, for single values either. (In this file at least)
3) It doesn't follow the order of other languages (not really a major concern).